src/services/currencyRates.js

```diff
@@ -0,0 +1,10 @@
+import { fetchWithTimeout } from '../utility';
+import { defaultCurrencyQuery } from '../utility/appConstants';
+
+const getCurrencyByQuery = (query = defaultCurrencyQuery) => {
+  return fetchWithTimeout(
+    `https://free.currconv.com/api/v7/convert?q=${query}&compact=ultra&apiKey=${process.env.REACT_APP_CURRENCY_API_KEY}`
```

Comment:
Move static URLs to `.env` file. And add it to `.env.sample` file to for reference.

Reply:
okay